File: beecrowd/beginner/c/1066.c
#include <stdio.h>

int main() {
    int n, count_neg, count_pos, count_odd, count_even;
    
    count_neg = count_pos = count_odd = count_even = 0;

    for (int i = 0; i < 5; i++) {
        scanf("%d", &n);
        
        if (n % 2 == 0){
            count_even++;
        } else {
            count_odd++; 
        }

        if (n < 0) {
            count_neg++;
        } else if (n != 0){
            count_pos++;
        }
    }

    printf("%d valor(es) par(es)\n", count_even);
    printf("%d valor(es) impar(es)\n", count_odd);
    printf("%d valor(es) positivo(s)\n", count_pos);
    printf("%d valor(es) negativo(s)\n", count_neg);

    return 0;
}

File: beecrowd/beginner/python/1066.py
even = positive = zeros = 0

for _ in range(5):
    n = int(input())

    if n % 2 == 0:
        even += 1
    if n > 0:
        positive += 1
    elif n == 0:
        zeros += 1

print(f'{even} valor(es) par(es)')
print(f'{5 - even} valor(es) impar(es)')
print(f'{positive} valor(es) positivo(s)')
print(f'{5 - zeros - positive} valor(es) negativo(s)')
